lib/job-manager.js

```diff
@@ -168,7 +168,9 @@ class JobManager {
   }
 
   killWorker (worker) {
-    worker ??= this.worker;
+    if (worker == null) {
+      worker = this.worker;
+    }
```

Comment:
Not 100% sure why this worked, I think it's babel doing its thing 🤔 

https://node.green/#ES2021-features-Logical-Assignment-----basic-support